src/types/interfaces.tsx

```diff
@@ -0,0 +1,21 @@
+export interface ParsedManualTags {
```

Comment:
Can you change this file to .ts extension as  it is not a tsx file

Reply:
changed

Reply:
removed